src/components/AppContainer/index.ts

```diff
@@ -0,0 +1,5 @@
+import styled from "styled-components";
+
+export const AppContainer = styled.div`
+  padding: 68px 10px 0;
```

Comment:
I'm like fairly sure I changed these values in one of my style commits 🤔check to make sure there are no conflicts

Reply:
Whoops, my bad